packages/account-v2/src/containers/DocumentSelection/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useBreakpoint } from '@deriv/quill-design';
+import { StandaloneChevronRightRegularIcon } from '@deriv/quill-icons';
+import { Text } from '@deriv-com/ui';
+import { Card } from '../../components/Card';
+import { getManualUploadDocumentList } from '../../constants/manual-document';
+
+type TDocumentSelection = {
+    countryCode: string;
+    handleOnClick: (value: string) => void;
+};
+
+const DocumentSelection = ({ countryCode, handleOnClick }: TDocumentSelection) => {
+    const { isMobile } = useBreakpoint();
+
+    const documentList = getManualUploadDocumentList(countryCode);
+
+    return (
+        <div className='flex flex-col'>
+            <Text as='h2' className='manual-poi__title' color='prominent' size='xs'>
+                Please upload one of the following documents:
+            </Text>
+            <div className='gap-900'>
+                {documentList.map(document => (
```

Comment:
document =>
`{ title, value, description, icon } =>`

Reply:
Sure will make this change